src/patternfly/components/InlineEdit/inline-edit.scss

```diff
@@ -0,0 +1,202 @@
+// Don't remove this magic comment. See gulpfile.js.
+// @import "../../sass-utilities/all";
```

Comment:
You can remove this. Also added an issue to remove it from the blueprint file https://github.com/patternfly/patternfly-next/issues/2454